styles/components/dropdown.scss

```diff
@@ -66,15 +64,33 @@
     // 4px = allowance
   }
 
-  // Dropup
+  > .menu > .divider {
+    height: 1px;
+    margin: 5px 0px;
```

Comment:
It can't be done with that. it will do this

![1](https://cloud.githubusercontent.com/assets/12628112/10784697/85f6807c-7d9a-11e5-9b1d-5cb6048ab53b.PNG)